forge-gui/res/cardsfolder/upcoming/warlords_elite.txt

```diff
@@ -0,0 +1,7 @@
+Name:Warlord's Elite
+ManaCost:2 W
+Types:Creature Human Soldier
+PT:4/4
+DeckHints:Type$Artifact
+A:SP$ PermanentCreature | Cost$ 2 W tapXType<2/Artifact;Creature;Land/artifacts, creatures, and/or lands> | CostDesc$ As an additional cost to cast this spell, tap two untapped artifacts, creatures, and/or lands you control. | SpellDescription$ As an additional cost to cast this spell, tap two untapped artifacts, creatures, and/or lands you control.
```

Comment:
@Northmoc @tool4ever should such thing "as an additonal cost to cast this spell" be written as RaiseCost instead?

Reply:
I know we haven't in the past... for stuff like Dargo and Gorex I think it has to be this way.

Reply:
But those are much more complicated... for this I think it could go either way?